assets/base.css

```diff
@@ -971,20 +971,20 @@ summary::-webkit-details-marker {
   display: flex;
   flex-wrap: wrap;
   margin-bottom: 2rem;
-  margin-left: calc(-1 * var(--grid-mobile-horizontal-spacing));;
   padding: 0;
   list-style: none;
+  column-gap: var(--grid-mobile-horizontal-spacing);
+  row-gap: var(--grid-mobile-vertical-spacing);
 }
 
 @media screen and (min-width: 750px) {
   .grid {
-    margin-left: calc(-1 * var(--grid-desktop-horizontal-spacing));
+    column-gap: var(--grid-desktop-horizontal-spacing);
+    row-gap: var(--grid-desktop-vertical-spacing);
```

Comment:
Two questions about the Product Media grid, specifically:
1) Mobile peekaboo is gone if we increase the horizontal spacing. I understand that the original goal was to improve discoverability of other media in the carousel, but wanted to confirm if this is how it should work with these new settings.

![image](https://user-images.githubusercontent.com/37168033/149009291-2e463506-674f-4eee-bdc3-c2e90888bc72.png)

2) This is probably a bug: if we increase the horizontal spacing, the product media / info layout will break. I believe the calculations for the `product__info-wrapper` and `product__media-wrapper` sizes are still based on the old `1rem` gap. We would need to update `section-main-product` to use the variable there too.

![image](https://user-images.githubusercontent.com/37168033/149009791-72c0d3e6-3a06-4004-be3b-0b6df7ce1ea7.png)

![image](https://user-images.githubusercontent.com/37168033/149010316-7cb99224-c050-464b-8d9d-15c6036048b4.png)

Reply:
Im wondering for `the product__info-wrapper` if I should remove any `padding-left` value it's currently receiving so that the spacing is only coming from the general setting 🤔 

I've made an update in this commit, let me know what you think: https://github.com/Shopify/dawn/pull/1127/commits/34f0638799fa1254ce7e786ce50360e8ce9135d8

Reply:
@melissaperreault , i've made it so sliders that have 1 item per page aren't applying any extra spacing to keep the peekaboo effect, is that ok ? I thought it made sense as there isn't anything really to gain from adding more spacing between those slides: [video](https://screenshot.click/22-01-idiet-7cky7.mp4)

Reply:
K so I tweaked some of the sliders work a bit better (scroll-padding-left update to match padding applied on the first slide). 

Made it so that horizontal space from the global settings will apply to sliders but it can cause this "issue": [video](https://screenshot.click/22-01-xpvxi-766mv.mp4)